awesomeConfig.cmake

```diff
@@ -352,6 +357,27 @@ foreach(file ${AWESOME_CONFIGURE_FILES})
                    ESCAPE_QUOTES
                    @ONLY)
 endforeach()
+
+set(AWESOME_CONFIGURE_NO_COV_FILES
+    ${awesome_c_configure_files}
+    ${awesome_lua_configure_files}
+)
+
+if(NOT DO_COVERAGE)
```

Comment:
Switch the `if` blocks to have the positive one first.
Also a comment here would help.